src/sonic-yang-models/yang-models/sonic-ntp.yang

```diff
@@ -112,10 +173,96 @@ module sonic-ntp {
                 leaf server_address {
                     type inet:host;
                 }
+
+                leaf association_type {
+                    type association-type;
+                    default server;
+                    description "NTP remote association type. Server, pool, or
+                                 peer.";
```

Comment:
Is `peer` allowed? I don't see any examples/handling for this.

Reply:
not allwoed, will fix description